scripts/load_test/app.py

```diff
@@ -0,0 +1,28 @@
+from fastapi import FastAPI, WebSocket
+from fastapi.responses import StreamingResponse
+import asyncio
+
+app = FastAPI()
+
+async def number_generator():
+    for number in range(1, 501):
+        message = "Lorem "*(number + 1)
+        yield f"data: {message}\n\n"
+        await asyncio.sleep(0.01)
+
+@app.get("/sse")
+async def sse():
+    return StreamingResponse(number_generator(), media_type="text/event-stream")
+
+@app.websocket("/ws")
+async def websocket_endpoint(websocket: WebSocket):
+    await websocket.accept()
+    for number in range(1, 501):
+        message = "Lorem "*(number + 1)
+        await websocket.send_text(message)
+        await asyncio.sleep(0.01)
```

Comment:
Nit: reduce duplication
```suggestion
    gen = number_generator()
    while True:
        try:
            message = await gen.__anext__()
            await websocket.send_text(message)
        except StopIteration:
            break
```